packages/one-trust-notices/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "pagopa-interop-one-trust-notices",
+  "main": "./dist/index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\"",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:autofix": "eslint . --ext .ts,.tsx --fix",
+    "format:check": "prettier --check src",
+    "format:write": "prettier --write src",
+    "start": "node --watch --loader ts-node/esm ./src/index.ts",
+    "build": "tsc",
+    "check": "tsc --project tsconfig.check.json"
+  },
+  "devDependencies": {
+    "@pagopa/eslint-config": "3.0.0",
+    "@types/html2json": "1.0.1",
+    "@types/lodash": "4.14.196",
+    "@types/node": "20.4.9",
+    "prettier": "2.8.8",
+    "ts-node": "10.9.2",
+    "typescript": "5.4.5"
+  },
+  "dependencies": {
+    "html2json": "^1.0.2",
+    "axios": "^1.7.4",
+    "dotenv": "^16.3.1",
+    "lodash": "^4.17.21",
```

Comment:
if you like, to reduce third party deps, it's possible to imports only isEmpty fn using proprly package 
 https://www.npmjs.com/package/lodash.isempty

Reply:
Thanks you!